addons/binding/org.openhab.binding.max/src/main/java/org/openhab/binding/max/internal/command/UdpCubeCommand.java

```diff
@@ -58,14 +58,14 @@ public UdpCubeCommand(UdpCommandType commandType, String serialNumber) {
 
     /**
      * UDP command types
-     * RESET - R Reset
+     * RESET - R Reboot
```

Comment:
Now it says reset - reboot?

Reply:
I clearly need to be more awake when doing some programming :(